packages/block-library/src/paragraph/style.scss

```diff
@@ -28,6 +28,13 @@ p {
 		text-transform: uppercase;
 		font-style: normal;
 	}
+
+	&.has-drop-cap::after {
```

Comment:
Should we apply this only when the RichText is focused? The dropcap is not shown when the block is focused. 

Right now, if you type in a block with DropCap you can see that it's height is not the regular height because of this style.

Reply:
If you mean apply only when not focussed, then yes I missed that, and added!

If you do mean focussed then I'm not sure - can you add a screenshot?